### ISSUE TITLE
Error in the documentation of tf.keras.losses.MeanAbsolutePercentageError

### ISSUE THREAD

**Issue by pedugnat** [keras-team/keras#15599]
# Description of the issue

The documentation of MAPE wrongly states that it is defined as `loss = 100 * abs(y_true - y_pred) / y_true` (see https://github.com/keras-team/keras/blob/2c48a3b38b6b6139be2da501982fd2f61d7d48fe/keras/losses.py#L372-L429).

However, it is implemented the right way (see this function, both documentation and computation are good : https://github.com/keras-team/keras/blob/2c48a3b38b6b6139be2da501982fd2f61d7d48fe/keras/losses.py#L1349)

# Proposed fix

Change `loss = 100 * abs(y_true - y_pred) / y_true` to `loss = 100 * abs((y_true - y_pred) / y_true)` to match the true formula used in computation. I can make the PR if you think it's relevant!

# Addendum

I also noticed that the implementation uses an epsilon in the denominator to avoid 0-division, which makes sense (same as in scikit-learn, for instance, see https://github.com/scikit-learn/scikit-learn/blob/0d378913b/sklearn/metrics/_regression.py#L286). I think it would be worthwhile to mention it also in keras' doc.

**Comment by sanatmpa1** [comment]
@pedugnat,

Thankyou for reporting the issue and I've submitted a [PR](https://github.com/keras-team/keras/pull/15604) to address the same.
